status/projects.py

```diff
@@ -397,6 +401,28 @@ def sample_list(self, project):
         return samples
 
 
+class FragAnImageHandler(SafeHandler):
+    """serves fragment_analyzer images based on their API uri"""
+
+    def get(self, project, sample, step):
+        self.set_header("Content-type", "application/json")
+        sample_view = self.application.projects_db.view("project/frag_an_links")
+        result = sample_view[project]
+        try:
+            data = result.rows[0].value
+        except TypeError:
+            #can be triggered by the data.get().get() calls.
+            self.write("no Fragment Analyzer image found")
+        else:
+            self.write(self.get_frag_an_image(data.get(sample).get(step)))
+
+    def get_frag_an_image(self,url):
+        data = lims.get_file_contents(uri=url)
+        encoded_string = base64.b64encode(data.read())
```

Comment:
```python
Uncaught exception GET /api/v1/frag_an_image/P9560/P9560_102/initial_qc (130.237.207.27)
    Traceback (most recent call last):
      File "/home/genomics.www/anaconda/envs/web/lib/python2.7/site-packages/tornado-4.0.2-py2.7-linux-x86_64.egg/tornado/web.py", line 1332, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/home/genomics.www/status/status/projects.py", line 417, in get
        self.write(self.get_frag_an_image(data.get(sample).get(step)))
      File "/home/genomics.www/status/status/projects.py", line 421, in get_frag_an_image
        encoded_string = base64.b64encode(data.read())
    AttributeError: 'unicode' object has no attribute 'read'
```

Reply:
This works with `genologics==0.3.15` and on a LIMS server where we actually have the project (not stage) 
<img width="566" alt="screen shot 2018-02-01 at 4 11 17 pm" src="https://user-images.githubusercontent.com/6576325/35685819-050a8758-076b-11e8-937f-c2bb30c21dc4.png">